src/common/model/createMemoryStore.test.ts

```diff
@@ -0,0 +1,72 @@
+import { MemoryVectorStore } from 'langchain/vectorstores/memory';
+import { CreateMemoryStore } from './createMemoryStore';
+import { OpenAIEmbeddings } from 'langchain/embeddings/openai';
+
+describe('CreateMemoryStore function', () => {
+  const initialFiles = [
```

Comment:
can you make this file in the testFiles and then read it here. Makes the code a lot nicer.

Reply:
done :)